torch/csrc/jit/script/compiler.cpp

```diff
@@ -806,6 +806,60 @@ std::shared_ptr<SugaredValue> BuiltinFunction::call(
       loc, *m.graph(), symbol, self, inputs, attributes, true));
 }
 
+std::shared_ptr<SugaredValue> ForkValue::call(
+    SourceRange loc,
+    Method& m,
+    const std::shared_ptr<SugaredValue> &forked,
+    NamedValue& input) {
+  auto inputs = toValues(*m.graph(), {input});
```

Comment:
I wouldn't make this a ::call metohd on ForkValue becuase it makes it seem like it is an overload of the normal call function. We can just make this a method of to_ir.

You can call `input.value(*m.graph())` there is no need to check that input size is 1 at this point.